src/svg/path/path.py

```diff
@@ -49,6 +49,31 @@ def __ne__(self, other):
             return NotImplemented
         return not self == other
 
+    def __add__(self, other):
+        self.start += other
+        self.end += other
+
+    def __sub__(self, other):
```

Comment:
And this is just a negative transform, really...

Reply:
It is, but also dunder methods would allow you to say:
`new_line = line - (6+3j)`

Which is generally easy to understand as a command as to what you'd want.